src/style.css

```diff
@@ -0,0 +1,124 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+  width: 200px;
+}
+
+.card {
+  width: 200px;
+  background-color: var(--color-white);
+  border: 1px solid var(--color-concrete);
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin-left: calc(50% - 160px / 2);
+  margin-top: 33px;
+  margin-bottom: 36px;
+}
+
+.card__box {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin-right: calc(50% - 166px / 2);
+  margin-left: calc(50% - 166px / 2);
+}
+
+.card__title-1 {
+  font-weight: 500;
+  font-size: 12px;
+  color: #060b35;
+  height: 36px;
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.card__title-2 {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  height: 14px;
+  margin-bottom: 16px;
+}
+
+.card__stars--4 {
+  height: 16px;
+  display: flex;
+}
+  
+.stars__star {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+  
+.stars__star:last-child {
+  margin-right: 0;
+}
+  
+.stars__star:nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
return here all this block from stars task. It has to work if you'll change this modifier in html
Reuse stars block from [Stars task](https://github.com/mate-academy/layout_stars) and keep it operational. If you change stars--4 modifier to stars--2 or other - number of selected stars should change correctly.